### PR TITLE
23.10 commits to develop

### PR DESCRIPTION
**Pull request by matejkriz**
## Description

Everything T2B1 related or 23.10 release related excluding final assets.

## Original PRs

All went through reviews there...

https://github.com/trezor/trezor-suite-private/pull/37
https://github.com/trezor/trezor-suite-private/pull/38
https://github.com/trezor/trezor-suite-private/pull/40
https://github.com/trezor/trezor-suite-private/pull/42
https://github.com/trezor/trezor-suite-private/pull/46
https://github.com/trezor/trezor-suite-private/pull/48
https://github.com/trezor/trezor-suite-private/pull/58
https://github.com/trezor/trezor-suite-private/pull/61
https://github.com/trezor/trezor-suite-private/pull/70

### CODE REVIEW

**Comment by matejkriz** [comment]
When we merge this, all that'll be left are those 4 commits: https://github.com/trezor/trezor-suite-private/pull/70

**Review by AdamSchinzel — APPROVED** [review]
lgtm

**Review by komret — APPROVED** [review]

**Comment by matejkriz** [comment]
t2b1-device-settings.test.ts.mp4 is failing (stucked on device authenticity check), I have to fix it before merge

File: packages/suite-web/e2e/tests/settings/t2b1-device-settings.test.ts [hunk 1]
```diff
@@ -43,11 +43,15 @@ describe('T2B1 - Device settings', () => {
...
         cy.prefixedVisit('/settings/device');
         cy.passThroughInitialRun();
+        cy.passThroughAuthenticityCheck();
 
         //
         // Test execution
         //
 
+        // cy.getTestElement('@suite/menu/settings').click();
+        // cy.getTestElement('@settings/menu/device').click();
```
Comment (matejkriz) [inline packages/suite-web/e2e/tests/settings/t2b1-device-settings.test.ts]: leftover, will remove

**Comment by matejkriz** [comment]
all greeen, merging...

**Pull request merged**